woocommerce/woocommerce-ios
# Block sign ups to WordPress.com accounts.

Closes #3146 

⚠️ Please note this PR is not against develop, but against the feature branch for UL&S 🙇

|  | Light | Dark |
| ---- | ---- | --- |
| Error screen | <img src="https://user-images.githubusercontent.com/2722505/101124246-98377200-3631-11eb-9e87-e440a80ff55f.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/101124182-776f1c80-3631-11eb-96f8-c15b91d54a76.png" width="350"/> |
| Error screen + alert | <img src="https://user-images.githubusercontent.com/2722505/101124176-763def80-3631-11eb-858b-7a17f1fc5b3c.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/101124178-76d68600-3631-11eb-9ca8-793fc55cc9ec.png" width="350"/> |

And a gif showing the interactions:
<img src="https://user-images.githubusercontent.com/2722505/101124157-6de5b480-3631-11eb-9b5f-81eb0e29537b.gif" width="350"/>

I am not sure if the actual link that can be opened in a web view from the alert is correct. Changing that takes 30 seconds.

## Changes
* Pointed WPAuthenticator to the branch that provides support for short-circuiting the navigation flow provided by WPAuthenticator, and allows us to block the account creation flow.
* A new implementation of the ULErrorViewModel that provides support for this new error screen.
* A couple of minor touches to the constraints in the actual UI, so that the button that is styled as a link can stretch horizontally to occupy all the available space.

## Testing
* Checkout the branch. Run `bundle exec pod install`
* Log out if necessary
* Select Continue With WordPress.com
* Enter an email address that does not match a WP.com account. 
* You should see the custom error UI instead of the flow that allows creating a new WP.com account



Update release notes:
